core/project/Dependencies.scala

```diff
@@ -3,9 +3,9 @@ import sbt._
 // format: OFF
 object Version {
 
-  val Akka          = "2.6.5"
-  val AkkaHttp      = "10.1.11"
-  val AkkaMgmt      = "1.0.6"
+  val Akka          = "2.6.6"
+  val AkkaHttp      = "10.1.12"
+  val AkkaMgmt      = "1.0.8"
```

Comment:
http and mgmt doesn't have to update, but I guess it will not hurt to use latest

Reply:
As long as it does not result in many versions of akka at runtime it's all good, but I don't have to explain that to you ;-) I'll run a test with this, thanks @patriknw!